stm32f2xx_src/stm32f2xx_can_drivers.c

```diff
@@ -0,0 +1,105 @@
+#include "can_raw.h"
+
+#include "stm32f2xx_hal_can.h"
+#include <stdint.h>
+
+static CAN_HandleTypeDef CanHandle;
+
+void Can_Init(uint32_t baudrate) {
+
+	// TODO calculate baudrate and set stuff accordingly
+    // Right now the baudrate is hard-coded to 500kbaud
+    // Can change prescaler to change this
+    // 2 Mbaud / prescaler = baudrate (prescaler goes from 1 to 1024)
+
+	CAN_FilterConfTypeDef  sFilterConfig;
+    static CanTxMsgTypeDef        TxMessage;
+    static CanRxMsgTypeDef        RxMessage;
+
+    CanHandle.Instance = CAN1;
```

Comment:
We need to think about support for multiple buses on the interface level. Make the CANlib great again!